pkg/services/authz/rbac/service_test.go

```diff
@@ -297,6 +297,75 @@ func TestService_checkPermission(t *testing.T) {
 	}
 }
 
+func TestService_mapping(t *testing.T) {
```

Comment:
I thought we may have issues here... but that was not the right path; but added a basic test seems good regardless so keeping it here